src/Grpc.Net.Client/Internal/OperatingSystem.cs

```diff
@@ -72,9 +72,9 @@ private OperatingSystem()
         //
         // Get correct Windows version directly from Windows by calling RtlGetVersion.
         // https://www.pinvoke.net/default.aspx/ntdll/RtlGetVersion.html
```

Comment:
Maybe add a note that RtlGetVersion isn't available in UWP - that check isn't very intuitive otherwise.